Sources/Widgets/SVG/SVGRepresentation/index.js

```diff
@@ -149,14 +142,19 @@ function vtkSVGRepresentation(publicAPI, model) {
   publicAPI.render = () => {
     throw new Error('Not implemented');
   };
+
+  /**
+   * Return the whole widget state.
+   */
+  publicAPI.getSelectedState = (prop, compositeID) => model.inputData[0];
```

Comment:
At some point do we want to try associating the prop/compositeID with the appropriate state? e.g. hovering over an SVG landmark rep to select the underlying handle state.

Reply:
Eventually I think it could, yes.
It does not make sense for the LineWidget (tested with) though.

Reply:
Ah, I understand now. You were right, I could use the default implementation. I just had to switch `prop` and `compositeId`